src/tests/ffi.cpp

```diff
@@ -4500,6 +4500,114 @@ TEST_F(rnp_tests, test_ffi_enarmor_dearmor)
     }
 }
 
+TEST_F(rnp_tests, test_ffi_customized_enarmor)
+{
+    // enarmor message
+    const std::string msg("this is a test long enough to have more than 72 characters in "
+                          "enarmored representation");
+    std::string       data;
+
+    // enarmor with line_length = 64
+    data.clear();
+    {
+        uint8_t *    buf = NULL;
+        size_t       buf_size = 0;
+        rnp_input_t  input = NULL;
+        rnp_output_t output = NULL;
+        rnp_output_t armor_layer = NULL;
```

Comment:
Personally I'd prefer to have same-named variable to be declared in single place, without repeating in the each block. Also additional spacing makes code slightly less readable. However this is just my personal opinion.
@dewyatt What do you think?

Reply:
I get why one would want to limit the scope of variables like this, but I'd agree with Nick that it's probably not worth it here, and the blank lines are a bit much.